test/cljs/rems/administration/test_create_catalogue_item.cljs

```diff
@@ -46,3 +46,82 @@
     (testing "missing form"
       (is (nil? (build-request (assoc form :form nil)
                                languages))))))
+
+(deftest resourse-dropdown-test
```

Comment:
nit: this should probably be `resource-label-test` to match the name of the function

Reply:
yeah, right, I missed it in the refactoring